src/app/addons/blog/data/posts/redwoodsdk-v1-beta.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "RedwoodSDK v1.0.0 Beta: A Major Milestone at the Edge 🚀"
+description: "We're excited to share that RedwoodSDK v1.0.0 Beta has officially landed! This marks an important step on our journey to a stable v1.0 release."
+date: "2025-10-08"
+author:
+  id: "herman"
+heroImage: "dfe10477-7c62-4481-49f6-edd20aa03d00"
+ogImage: "https://imagedelivery.net/EBSSfnGYYD9-tGTmYMjDgg/4b7a124b-6a74-4d9b-aade-b26871b30500/public"
+tags: ["redwoodsdk", "cloudflare", "v1.0.0", "beta"]
+---
+# RedwoodSDK v1.0.0 Beta: A Major Milestone at the Edge 🚀
```

Comment:
Bikeshed: `v1.0.0 Beta` is technically correct, but I'm thinking `1.0 Beta` maybe rings better. Wdyt?